project/project.go

```diff
@@ -65,15 +65,23 @@ func NewProject(context *Context, runtime RuntimeProject, parseOptions *config.P
 	}
 
 	if context.EnvironmentLookup == nil {
-		cwd, err := os.Getwd()
+		var envPath, absPath, dir string
+		var err error
+		for _, file := range context.ComposeFiles {
```

Comment:
We should just look at the first Compose file. This will always look at the last.